helpful.el

```diff
@@ -1036,18 +1085,13 @@ state of the current symbol."
       (t
        "Could not find source file."))
      "\n\n"
-     (helpful--button
-      "Find all references"
-      'helpful-all-references-button
-      'symbol helpful--sym
-      'callable-p helpful--callable-p))
+     (helpful--make-references-button "Find all references"))
 
     (when (helpful--advised-p helpful--sym)
       (helpful--insert-section-break)
       (insert
        (helpful--heading "Advice")
-       (format
-        "This %s is advised." (if (macrop helpful--sym) "macro" "function"))))
+       (format "This %s is advised." (downcase sym-type))))
```

Comment:
I think "this function is advised" is a little clearer than "this command is advised", so I'd prefer to leave the text as-is here.

Reply:
`sym-type` should evaluate to `"function"` or `"macro"`, so this wouldn't change anything. Or maybe I don't understand what you mean.

Reply:
Woops! You're right -- I thought `sym-type` could have the value `"Command"`, but I was mistaken.